hack/make-rules/test.sh

```diff
@@ -166,6 +170,10 @@ if [[ -n "${KUBE_RACE}" ]] ; then
   goflags+=("${KUBE_RACE}")
 fi
 
+if [[ -n "${KUBE_TESTCOUNT}" ]] ; then
+  goflags+=("-count=${KUBE_TESTCOUNT}")
```

Comment:
A thought: Why not just `KUBE_TEST_GOFLAGS` instead of a special env for each flag? `KUBE_RACE=-race` already feels bad, and if I want to pass some other flag to unit tests only (and not any other compilation we do under that make call) I have to add another change like this.

Reply:
by ignorance, basically :)